graphics/gscan2pdf/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           perl5 1.0
+
+name                gscan2pdf
+version             2.12.4
+revision            0
+
+categories          graphics
+platforms           darwin
```

Comment:
this is the default since MacPorts v2.7 so the line can be removed (also in the other ports)

Reply:
please remove that line in every Portfile in this PR

Reply:
done, haven't seen them